From 495ee430783a8a0566c45ac33a0e7b2e5443b0fe Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 25 Jul 2013 12:48:55 +0000
Subject: [PATCH 0449/1077] mControl.exe V1.0.2.11 Mantis  0028216: mcontrol:
 do not create logging client before logging servant is started

git-svn-id: svn://localhost/SelfServiceCommon/trunk@546 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mcontrol_def.h          | 11 ++++++++---
 .../Massai/cpp/Controller/src/serverwin.cpp           |  3 +--
 2 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
index 6181fdcc..951fd452 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
@@ -9,17 +9,22 @@
  *
  ************************************************************************/
 
-#define FILE_VER_BIN     1,0,2,10
-#define FILE_VER_STR     "FileVersion", "1.0.2.10\0"
+#define FILE_VER_BIN     1,0,2,11
+#define FILE_VER_STR     "FileVersion", "1.0.2.11\0"
 
 /*! \page mControl.exe - platform startup, starts mudpate.exe, mstarter.exe; servant for systemControl interface
 
-@version 1.0.2.9
+@version 1.0.2.11
 
 \section change1 Change history
 
+\date 25.07.2013
+\li V1.0.2.11
+\li bugfix - SteT: Mantis  0028216: mcontrol: do not create logging client before logging servant is started
+
 \date 25.07.2013 \li V1.0.2.10
                  \li GIT: Mantis0028214: mcontrol: Windows7; mcontrol.reboot is written local appdata
+\li bugfix - SteT: Mantis  0028216: mcontrol: do not create logging client before logging servant is started
 
 \date 22.07.2013
 \li V1.0.2.9
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
index ace797a2..446b2bd6 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
@@ -144,6 +144,7 @@ void waitInitialForLoggingService()
       TRACE(HERE,"waitInitialForLoggingService failed: unknown exception");
     }
     TRACE(HERE,"waitInitialForLoggingService done.");
+    createAlert();
   }
   init= true;
 }
@@ -656,8 +657,6 @@ LRESULT CALLBACK serverwinWindowProc
      case WM_CREATE:
           TRACE(SecondaryRuntime_HERE,"WM_CREATE");
 
-          createAlert();
-
           iccex.dwSize = sizeof(INITCOMMONCONTROLSEX);
           iccex.dwICC  = ICC_PROGRESS_CLASS;
 
-- 
2.41.0.windows.1

